try_install_all.py

```diff
@@ -24,7 +24,8 @@
     ]
 SCRIPT_LIST = SCRIPT_LIST()
 TEST_ENGINES = {}
-IGNORE = ["AvianBodyMass", "FIA", "Bioclim", "PRISM", "vertnet"]
+IGNORE = ["AvianBodyMass", "FIA", "Bioclim", "PRISM", "vertnet","NPN", "mammsupertree", "eBirdOD"]
```

Comment:
I agree that we shouldn't try to get eBird and and NPN working immediately. What I'm suggesting is preventing them from erroring and providing a useful message instead. My concern with ignoring them here is that they actually are failing and we want the testing to fail as well until they aren't throwing errors. Once tests get turned off they tend to get forgotten and not turned back on because they aren't throwing errors. Happy to chat more about this in person tomorrow so that we can figure out what's best.